src/FormatNumber.elm

```diff
@@ -26,6 +48,42 @@ type alias Locale =
     }
 
 
+
+-- Locales from
```

Comment:
Totally agree. You got me 💃 🕺 

Can you add these bits to the documentation? Your idea is good, let's make it more evident! (Or is it already there and I missed it?)